docs/docusaurus/docs/capture.md

```diff
@@ -172,30 +129,31 @@ CREATE TABLE fingerprint (mac TEXT NOT NULL, protocol TEXT, fingerprint TEXT, ti
 
 The timestamp is given as 64 bit microseconds value and the fingerprint string is encoded in base64.
 
-
 An example of the fingerprint table rows are below:
 
-|MACPROTOCOL|FINGERPRINT|TIMESTAMP|QUERY|
-|-----------|-----------|---------|-----|
-|84:e3:42:3a:cb:2f|TLS.Amazon|mI1ENXMPBQDVjwGh/o0bLSrD8+O2O5RCFQLbUVt4lzI|1625055051481102|a2.tuyaeu.com|
-|9c:ef:d5:fd:db:56|TLS.Amazon|mI1ENXMPBQDVjwGh/o0bLSrD8+O2O5RCFQLbUVt4lzI|1625055051481102|a2.tuyaeu.com|
-|84:e3:42:3a:cb:2f|DNS|NqRTRWiNdfG4zMkiXE9P0eRQIefPgMYV/vXUymxdvNw|1625055072748967|1.0.0.10.in-addr.arpa|
-|9c:ef:d5:fd:db:56|DNS|NqRTRWiNdfG4zMkiXE9P0eRQIefPgMYV/vXUymxdvNw|1625055072748967|1.0.0.10.in-addr.arpa|
-
+| MACPROTOCOL       | FINGERPRINT | TIMESTAMP                                   | QUERY            |
+| ----------------- | ----------- | ------------------------------------------- | ---------------- | --------------------- |
+| 84:e3:42:3a:cb:2f | TLS.Amazon  | mI1ENXMPBQDVjwGh/o0bLSrD8+O2O5RCFQLbUVt4lzI | 1625055051481102 | a2.tuyaeu.com         |
```

Comment:
I'm guessing `PROTOCOL` should be another column?

```suggestion
| MAC               | PROTOCOL    | FINGERPRINT                                 | TIMESTAMP        | QUERY                 |
| ----------------- | ----------- | ------------------------------------------- | ---------------- | --------------------- |
| 84:e3:42:3a:cb:2f | TLS.Amazon  | mI1ENXMPBQDVjwGh/o0bLSrD8+O2O5RCFQLbUVt4lzI | 1625055051481102 | a2.tuyaeu.com         |
```

Reply:
Hey @mereacere, just for next time, there's a **Commit suggestion** button on the bottom right of suggestions on PRs :)

![image](https://user-images.githubusercontent.com/19716675/162695458-6d3e637e-c896-4b87-a777-357613a259f0.png)


It makes it a lot easier if all the comments in a PR are resolved within the same PR, instead of us having to make another commit/PR and have to split up the discussion into multiple places.